content/en/opentelemetry/collector_exporter/_index.md

```diff
@@ -0,0 +1,222 @@
+
+{{< img src="metrics/otel/datadog_exporter.png" alt="Application Instrumented Library, Cloud Integrations, and Other Monitoring Solutions (for example Prometheus) -> Datadog Exporter inside OpenTelemetry Collector -> Datadog" style="width:100%;">}}
+
+
+## Setting up the OpenTelemetry Collector with the Datadog Exporter
+
+To run the OpenTelemetry Collector along with the Datadog Exporter, download the latest release of the [OpenTelemetry Collector Contrib distribution][3]. 
+
+### Out-of-the-box Datadog Exporter configuration
+
+In the `exporter/datadogexporter` folder is a working example of an out-of-the-box configuration for Datadog Exporter. See the full configuration example file, [`ootb-ec2.yaml`][4]. Configure each of the following components to suit your needs:
+
+{{< whatsnext desc="Exporter components" >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/hostname_tagging/" >}}Hostname and tags{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/otlp_receiver/" >}}OTLP Receiver{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/host_metrics/" >}}Host Metrics{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/docker_metrics/" >}}Docker Metrics{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/log_collection/" >}}Log Collection{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/collector_health_metrics/" >}}Collector Health Metrics{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/trace_metrics/" >}}Trace Metrics{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/collector_batch_memory/" >}}Collector Batch and Memory Settings{{< /nextlink >}}
+{{< /whatsnext >}}
+
+
+## Running the collector
+
+{{< tabs >}}
+{{% tab "On a host" %}}
+
+Run the collector, specifying the configuration file using the `--config` parameter:
+
+```
+otelcontribcol_linux_amd64 --config collector.yaml
+```
+
+{{% /tab %}}
+
+{{% tab "Docker (localhost)" %}}
+To run the OpenTelemetry Collector as a Docker image and receive traces from the same host:
+
+1. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib`][1].
+
+2. Determine which ports to open on your container so that OpenTelemetry traces are sent to the OpenTelemetry Collector. By default, traces are sent over gRPC on port 4317. If you don't use gRPC, use port 4318.
+
+3. Run the container and expose the necessary port, using the previously defined `collector.yaml` file. For example, considering you are using port 4317:
+
+   ```
+   $ docker run \
+       -p 4317:4317 \
+       --hostname $(hostname) \
+       -v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml \
+       otel/opentelemetry-collector-contrib
+   ```
+
+
+[1]: https://hub.docker.com/r/otel/opentelemetry-collector-contrib/tags
+{{% /tab %}}
+{{% tab "Docker (other containers)" %}}
```

Comment:
where does configuration go? i see an `otel_collector_config.yaml` and a `config.yaml` in step 2, is that related? if so, which one/both?

Reply:
Are you referring to `-v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml` ? If so, this is to mount volumes, so the `otel_collector_config.yaml` on the machine will be available in the container at this path: `/etc/otelcol-contrib/config.yaml`.